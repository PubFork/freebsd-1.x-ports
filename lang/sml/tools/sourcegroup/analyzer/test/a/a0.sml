group ["a1.sml", "a2.sml"];
group ["a1.sml", "a3.sml"];
group ["a1.sml", "a4.sml"];
