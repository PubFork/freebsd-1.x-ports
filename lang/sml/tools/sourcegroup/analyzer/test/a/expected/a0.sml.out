source sml "a/a0.sml";

